weave-js/src/components/PagePanelComponents/Home/Browse3/pages/PlaygroundPage/usePlaygroundState.ts

```diff
@@ -106,6 +112,20 @@ export const usePlaygroundState = () => {
         if (inputs.presence_penalty) {
           newState.presencePenalty = parseFloat(inputs.presence_penalty);
         }
+        if (inputs.model) {
```

Comment:
loads the model into the state now, with a smart default if the model used isnt one of the one we support